app/presenters/api_docs/api_reference.rb

```diff
@@ -39,12 +40,26 @@ def field_lengths_summary
       end
     end
 
+    def self.draft_schema
```

Comment:
not sure why we needed to make those class methods?

Reply:
The idea was to load and memoize once on application startup/class loading rather than for every instance as these schemas won't change without deployment. Happy to make instance methods though, probably negligible CPU involved.